grass7/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.py

```diff
@@ -56,6 +66,11 @@
 #% guisection: Settings
 #%end
 #%flag
+#% key: o
+#% description: Override projection check
```

Comment:
```suggestion
#% description: Override projection check (use current location's projection)
```
... changed to standard wording (see e.g. r.import)